CMakeLists.txt

```diff
@@ -65,6 +65,30 @@ else ()
         CACHE STRING "Installation path for plugins" FORCE)
 endif ()
 
+if ( NOT ZEEK_LOCAL_STATE_DIR )
```

Comment:
Not sure if this is terminology we're already using, but at least from this code here I find the distinction between `ZEEK_LOCAL_STATE_DIR` and `ZEEK_STATE_DIR` confusing; the former doesn't sound like it would be the root directory. Could whatever is using `ZEEK_LOCAL_STATE_DIR` just use `ZEEK_ROOT_DIR` instead, or do we want to control that separately?